src/runtime/virtcontainers/virtiofsd.go

```diff
@@ -79,6 +79,10 @@ func (v *virtiofsd) getSocketFD() (*os.File, error) {
 		return nil, err
 	}
 
+	if err := utils.ChownToParent(v.socketPath); err != nil {
```

Comment:
`virtiofsd` is still running as root. We briefly discussed this a few weeks ago in the AC meeting. It would be nice to also run `virtiofsd` as a non-root process, but we agreed that's a separate task and can be triaged separately.

Reply:
ok, if that's the case, would you mind filing a bug and commenting it in the code

Reply:
https://github.com/kata-containers/kata-containers/issues/2542 created. I'll add a comment in the code as well.